modules/ob-configuring-histogram-buckets.adoc

```diff
@@ -0,0 +1,36 @@
+// This module is included in the following assembly:
+//
+// builds/build-controller-observability.adoc
+
+:_content-type: PROCEDURE
+[id="ob-configuring-histogram-buckets_{context}"]
+= Configuring a histogram bucket
+
+For the build controller, you can configure a histogram bucket by setting the value of the environment variable. When running the build controller locally, set the value of the environment variable right before starting it.
+
+When deploying the build controller on the {ocp-product-title} cluster, you must add the environment variable in the `controller.yaml` deployment file. The environment variable values must be a comma-separated list of numbers.
+
+.Prerequisites
+
+*
```

Comment:
@apoorvajagtap, Do we have any prerequisites for this section?

Reply:
I don't think we have any. @jkhelil can you please confirm on this?